pwiz_aux/msrc/utility/vendor_api/Waters/WatersRawFile.hpp

```diff
@@ -425,8 +425,41 @@ struct PWIZ_API_DECL RawData
         }
     }
 
+    void EnableDDAProcessing()
+    {
+        DDAProcessor.SetRawReader(Reader);

```

Comment:
Does this block execution for a significant amount of time?

Reply:
No, I believe not in this case, it just takes the handle from the existing reader.